plugins/scaffolder-common/api-report.md

```diff
@@ -116,6 +118,12 @@ export interface TemplateEntityV1beta3 extends Entity {
 export const templateEntityV1beta3Validator: KindValidator;
 
 // @public
+export type TemplateInfo = {
+  name: string;
```

Comment:
is it time to even make this `templateEntityRef: string` while we're making the larger overhaul?

Reply:
Mm. It's just that now it'll be two migrations so to speak :) aight

Reply:
But maybe it's fine - and I can make my next PR's a little bigger 👍

Reply:
Agree with freben here tbh, we'll be making this an unnecessary double migration otherwise right? We can keep the existing metadata as is and parse out the name from the ref, or if only name is provided we can stringify that with default namespace